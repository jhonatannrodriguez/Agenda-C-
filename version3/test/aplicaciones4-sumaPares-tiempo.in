sumaParesTiempo 20 20000 2

sumaParesTiempo 1000 20000 2

sumaParesTiempo 2 20000 2

sumaParesTiempo 20000 20000 2

sumaParesTiempo 1000000 30000 2

Fin