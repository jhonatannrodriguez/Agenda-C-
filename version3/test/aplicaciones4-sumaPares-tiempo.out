1>Existe un par de elementos en el conjunto cuya suma es 20.
Bien.
2>Existe un par de elementos en el conjunto cuya suma es 1000.
Bien.
3>No existe un par de elementos en el conjunto cuya suma es 2.
Bien.
4>Existe un par de elementos en el conjunto cuya suma es 20000.
Bien.
5>No existe un par de elementos en el conjunto cuya suma es 1000000.
Bien.
6>Fin.
